### ISSUE TITLE
integration with openproject

### ISSUE THREAD

**Issue by pow-nico** [calcom/cal.com#1742]
### Is your proposal related to a problem?

I would like to use cal.com together with https://www.openproject.org

### Describe the solution you'd like

Maybe it's possible to integrate their API in your product

### Describe alternatives you've considered

Nextcloud with Nextcloud Appointments App (https://github.com/SergeyMosin/Appointments)

Thanks for your consideration!

**Comment by PeerRich** [comment]
hey @nico-pwn what exactly do you want to integrate with of openproject?

**Comment by nico-pwn** [comment]
Hey @PeerRich,

as I understood cal.com has support for calDAV and some proprietary APIs for Apple, Microsoft and Google to integrate the appointments made in cal.com with these respective services.

Unfortunately openproject doesn't support calDAV and that's why I thought to open that feature request.

TLDR: integration of appointments of cal.com into openproject through their API (https://www.openproject.org/docs/api/)

**Comment by PeerRich** [comment]
> Hey @PeerRich,
> 
> 
> 
> as I understood cal.com has support for calDAV and some proprietary APIs for Apple, Microsoft and Google to integrate the appointments made in cal.com with these respective services.
> 
> 
> 
> Unfortunately openproject doesn't support calDAV and that's why I thought to open that feature request.
> 
> 
> 
> TLDR: integration of appointments of cal.com into openproject through their API (https://www.openproject.org/docs/api/)

do you know someone working on OP? happy to accept pull requests but we will likely not work on an integration ourselves

**Comment by pow-nico** [comment]
Not as I know. But I thought cal.com is somehow an extension of other established systems like calendars and project management tools: Calendar and project management works without cal.com but cal.com without the others makes not so much sense in my opinion.

**Comment by PeerRich** [comment]
i dont really see how it would integrate right now. can you explain which API we should connect? or maybe figma or something

**Comment by pow-nico** [comment]
OK, sorry for not being precise enough.

I thought about general project management. To organize a project you need all task which needs to be done to realize a project. Most of the time projects have deadlines. To achieve these deadlines you need to organize and priorities the different tasks in a project. 

If you are working together in a team or with other (external) persons you have to manage your time: When do I work on my project tasks? When do I meet or communicate with other people?

In the end you need to schedule your appointments with people and also your time working on tasks. There the integration of cal.com into open project starts.

That's why I would find it rather useful to get appointments from cal.com pushed to a board inside of open project as a task with a deadline ("Meet with John Doe at Tuesday 5 pm").

Here is an overview of the different kinds of boards in open project (not only kanban styled ones): https://www.openproject.org/docs/user-guide/agile-boards/

I hope that helps to understand. If you have any more questions I'm very welcome to answer.

**Comment by PeerRich** [comment]
i see. this is very low priority for us but happy to accept PRs that send bookings to the kanban board

**Comment by PeerRich** [comment]
if anyone wants to pick this up, here's how to make an app: https://docs.cal.com/how-to-guides/how-to-build-an-app

**Comment by PeerRich** [comment]
closing due to stale
